docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1485,47 +1485,53 @@ The ``urlsplit`` filter extracts the fragment, hostname, netloc, password, path,
 Searching strings with regular expressions
 ------------------------------------------
 
-To search a string with a regex, use the "regex_search" filter::
+To search in a string or extract parts of a string with a regular expression, use the ``regex_search`` filter::
 
-    # search for "foo" in "foobar"
-    {{ 'foobar' | regex_search('(foo)') }}
-    # => "foo"
+    # Extracts the database name from a string (returns "database42")
+    {{ 'server1/database42' | regex_search('database[0-9]+') }}
```

Comment:
```suggestion
    {{ 'server1/database42' | regex_search('database[0-9]+') }}
    # => "database42"
```

Reply:
I would keep the return output since other places it is the same.

Reply:
Indeed, thanks, I've updated the PR. Since the file inconsistently uses both `"` and `'`, I decided to stick to `'` because of the existing example output for `regex_replace`.